R/output_format.R

```diff
@@ -103,10 +103,7 @@ merge_post_processors <- function (base, overlay) {
 merge_output_formats <- function(base, overlay)  {
   structure(list(
     knitr = merge_lists(base$knitr, overlay$knitr),
-    pandoc = pandoc_options(
-      to = merge_scalar(base$pandoc$to, overlay$pandoc$to),
-      from = merge_scalar(base$pandoc$from, overlay$pandoc$from),
-      args = c(base$pandoc$args, overlay$pandoc$args)),
+    pandoc = merge_lists(base$pandoc, overlay$pandoc),
```

Comment:
The new behavior does not seem to be equivalent to the old one when dealing with `args`, so it may be a breaking change, which may be reasonable or not. I'm not sure. Cc @jjallaire